Backend/Controllers/EnemyController.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Backend.Services;
+using Domain.Entities;
+using Microsoft.AspNetCore.Cors.Infrastructure;
+
+namespace Backend.Controllers
+{
+    public static class EnemyController
+    {
+        public static void MapEnemyEndpoint(this WebApplication app)
+        {
+            //Get all enemies
+            app.MapGet("/enemies", (IEnemyService enemyService) =>
+            {
+                return enemyService.GetAllEnemies();
+            });
+
+            //Get enemy by id
+            app.MapGet("/enemies/{id}", (IEnemyService enemyService, int id) =>
+            {
+                return enemyService.GetEnemyById(id);
+            });
+
+            //Delete enemy
+            app.MapDelete("/enemies/{id}", (IEnemyService enemyService, int id) =>
+            {
+                enemyService.DeleteEnemy(id);
```

Comment:
Vi skal sørge for at der ikke kan gå ting galt undervejs når vi kører programmet. Da vi sad og kiggede på det sammen, så fejlede det hvis man slettede den samme 2 gange.
plz implementer at den ikke kommer til at fejle ved det